src/codeflare_sdk/cluster/cluster.py

```diff
@@ -1010,18 +959,14 @@ def _map_to_ray_cluster(rc) -> Optional[RayCluster]:
 
 
 def _map_to_app_wrapper(aw) -> AppWrapper:
-    if "status" in aw and "canrun" in aw["status"]:
+    if "status" in aw:
         return AppWrapper(
             name=aw["metadata"]["name"],
-            status=AppWrapperStatus(aw["status"]["state"].lower()),
-            can_run=aw["status"]["canrun"],
-            job_state=aw["status"]["queuejobstate"],
+            status=AppWrapperStatus(aw["status"].get("phase", "suspended").lower()),
         )
     return AppWrapper(
         name=aw["metadata"]["name"],
-        status=AppWrapperStatus("queueing"),
-        can_run=False,
-        job_state="Still adding to queue",
+        status=AppWrapperStatus("suspended"),
```

Comment:
Can you just use AppWrapperStatus.SUSPENDED directly here in order to avoid magic string?

Reply:
Done.